pkgs/development/python-modules/astroid/1.6.nix

```diff
@@ -0,0 +1,6 @@
+{ callPackage }:
```

Comment:
If 1.6 is a legacy version we need for Python 2, then we copy in the whole expression, as we don't know how much the new one will deviate from the old one.